sources/packages/web/src/components/aest/students/assessment/AssessmentAwardDetails.vue

```diff
@@ -0,0 +1,271 @@
+<template>
+  <div v-if="assessmentAwardData.estimatedAward">
+    <!-- Show the first payment/second payment header only if there is second disbursement. -->
+    <div v-if="isSecondDisbursementAvailable" class="font-weight-bold my-3">
```

Comment:
I believe that we have been trying to use `<p>` or `<h?>` for text to be more aligned and meaningful for accessibility.
Same for others in this file.